### PR TITLE
Add Project UUID as a column to the CaaS Job Manager

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose
_Please explain the purpose of this PR and include links to any GitHub issues that it fixes:_

- https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/484.
---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- Add project-uuid as a new column to the CaaS Job Manager.
- Make the failure analysis script more corner-case resistant. Right now it will run into errors when there are missing keys.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- No instructions.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: deploy/job-manager/deploy.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ function configure_kubernetes() {
 
     stdout "Setting to use GKE cluster: project gke_${GCLOUD_PROJECT}_us-central1-b_lira"
     kubectl config use-context ${GKE_CONTEXT}
+    kubectl config set-context $(kubectl config current-context) --namespace=green-100-us-central1-ns
```
[the remark this answers is not in the document]
Comment (rexwangcc) [reply deploy/job-manager/deploy.sh]: Done!
